index.js

```diff
@@ -123,7 +123,7 @@ function * createChildrenContent (view, options, hooks) {
 
 function * render (view, attrs, options) {
   options = options || {}
-  if (view.view) { // root component
+  if (view.view || isClass(view)) { // root component
```

Comment:
for root
https://github.com/MithrilJS/mithril-node-render/pull/72/commits/441385e89a6486e3a52a2545b679dd57881cc0c1#diff-1dd241c4cd3fd1dd89c570cee98b79ddR145